### PR TITLE
feat: 修改 get command 的query 参数

### PR DESCRIPTION
**Pull request by Ambition-J**
Get command 里面 ds-pushevent and ds-returnevent 的query parameters，值改为true or false

### CODE REVIEW

File: web/package-lock.json
Comment (ajcasagrande) [inline web/package-lock.json:1]: package-lock.json has been downgraded to version 1 from version 2. Please upgrade your npm and re-generate the lock file.

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]
there are too many unlrelated changes
please rebase the code

**Comment by cloudxxx8** [comment]
@Ambition-J are you able to rebase this PR?

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**